content/livesync/index.textile

```diff
@@ -0,0 +1,43 @@
+---
+title: What is LiveSync?
+meta_description: "LiveSync enables you to synchronize changes in your database to frontend clients at scale."
+product: livesync
+---
+
+LiveSync enables you to synchronize changes in your database to frontend clients at scale. It integrates with your database of choice and leverages Ably's global message delivery network to sync data between your users in realtime.
+
+LiveSync is made up of two key components:
+
+- Database Connector := A backend service for broadcasting changes from your database in realtime.
+- Models SDK := A frontend library that simplifies subscribing to changes in data models and applying optimistic updates, merging them with confirmed updates.
+
+The following diagram provides a simplified overview of the LiveSync solution:
+
+h2(#db-connector). Database Connector
+
+The "Database Connector":/livesync/connector uses the "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html to broadcast application-specific change events from your database to frontend clients via "Ably Channels":/channels.
+
+When you update the data in your database, you record a change event transactionally in a dedicated "outbox table":https://docs.google.com/document/d/19BJ4oYC3ah4wFH0VS5Qv6qHe3qmRS_PpNLSytTOi9tc/edit#heading=h.r00ywdkts746. The Database Connector detects new changes in the outbox and publishes them as messages on Ably Channels. Using the Models SDK, your client applications subscribe to these change events and update their local state with the new data.
```

Comment:
This shouldn't link to the google doc, but rather to the outbox table section on the docs site

Reply:
Fixed in this [commit](https://github.com/ably/docs/pull/2082/commits/15a2be26866049471006c24c9e1e9094c5b53a85)